configs/pytorch.json

```diff
@@ -1,8 +1,14 @@
 {
   "index_name": "pytorch",
   "start_urls": [
+    "https://pytorch.org",
```

Comment:
Adding this link will broader the crawl to https://pytorch.org/docs/stable/* pages that are powered by read the docs. Do you want the search to encompass such website or is it an undesired behaviour?

Reply:
Yeah, I can remove it.